api/types/mount/mount.go

```diff
@@ -56,3 +61,11 @@ type Driver struct {
 	Name    string            `json:",omitempty"`
 	Options map[string]string `json:",omitempty"`
 }
+
+// TmpfsOptions defines options specific to mounts of type "tmpfs".
+type TmpfsOptions struct {
+	// RawOptions is the raw string passed to mount(2).
+	// e.g. "rw,noexec,nosuid,size=65536k"
+	// RawOptions cannot contain "ro,", currently.
+	RawOptions string `json:",omitempty"`
```

Comment:
It is easy to support readonly tmpfs, but needs consideration for `m.ReadOnly==true && m.TmpfsOptions.RawOptions.contains("rw")`.
Not sure actually there have been people using readonly tmpfs (theoretically it can be used for hiding some files without deletion)